apps/web/src/application/services/CreateTransaction/CreateTransactionDtos.ts

```diff
@@ -1,5 +1,6 @@
 export interface CreateTransactionRequest {
   txType: string
   address: string
+  network: string
```

Comment:
I see there's a `NetworkEnum` - should then this be a type of that enum?

```suggestion
  network: NetworkEnum
```

Reply:
oh, I guess this comes from the FE, so it needs some validation at this point. If that's the case, then it is o

Reply:
Exactly. This comes from the front-end.